src/ui/public/courier/fetch/request/__tests__/segmentedCreateQueue.js

```diff
@@ -0,0 +1,62 @@
+describe('ui/courier/fetch/request/segmented/_createQueue', () => {
```

Comment:
I tried adding this to the existing test file but I was having OCD about the helper functions at the bottom and figured it didn't hurt to start a new file :neckbeard: 

As for the tests for `_createQueue()`, I rationalized this because in this specific instance I was using the `_` prefix to designate that this was not a method required for request objects. In other languages the inverse might be noted by adding `(override)` to the methods that had a specific contract. That said, the reason I didn't go for an overall test was because it felt too daunting and out-of-scope for the changes that I needed to verify...

If I was to fix this I would probably convert `_createQueue()` into an `IndexQueue` class that i could then test in isolation.